EIPS/eip-2031.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 2031
+title: State Rent B - Net transaction counter
+author: Alexey Akhunov (@AlexeyAkhunov)
+discussions-to: https://ethereum-magicians.org/t/eip-2031-net-transaction-counter-change-b-from-state-rent-v3-proposal/3283
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-05-15
+---
```

Comment:
Need to add `requires: 2029`.

Reply:
Thank you, done

Reply:
Thought it fails the check now, because it assumes that 2029 needs to be merged

Reply:
Yes, we should merge 2029 first.